src/sagemaker/instance_types_gpu_info_config/	af-south-1.json

```diff
@@ -0,0 +1,28 @@
+{
+    "registries": {
```

Comment:
Is there a whitespace in the filename here?

Reply:
Shouldn't have the whitespace. Will remove it.